draft-ietf-webtrans-http3.md

```diff
@@ -329,6 +329,20 @@ requests for new WebTransport sessions.  A GOAWAY frame does not affect data
 streams for existing WebTransport sessions; those can continue to be opened
 even after the GOAWAY frame has been sent or received.
 
+To drain a WebTransport session, either endpoint can send a
+DRAIN_WEBTRANSPORT_SESSION capsule.  An endpoint SHOULD NOT initiate new
+WebTransport streams or datagram flows after sending or receiving a
```

Comment:
"Datagram flows" is not a well-defined term. Is the intent to discourage all datagrams?

Reply:
I agree it's not well defined.  The intent was to prevent the application from starting new work over this session, but to allow it to finish existing work.  I don't think we should prevent sending all datagrams -- @DavidSchinazi do you have any text suggestions?

Reply:
Maybe use the text about existing work vs new work then?

Reply:
I'm not actually sure we should give specific guidance here.  "Stop opening new streams" makes sense for HTTP/3, but different protocols may have different drain sequence.  We should probably just describe what it means.

Reply:
Maybe something simple like:

```
To drain a WebTransport session, either endpoint can send a DRAIN_WEBTRANSPORT_SESSION capsule.  
After sending or receiving a DRAIN_WEBTRANSPORT_SESSION capsule, an endpoint MAY continue using the session but SHOULD attempt to gracefully terminate the session as soon as possible.
```

Reply:
Comment from editors -- replace endpoint with client to make it clear there is no action from the user-agent.